src/tests/Regressions/coreclr/GitHub_87879_AppHost/test87879.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Needed for mechanical merging of all remaining tests, this particular project may not actually need process isolation -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UseAppHost>true</UseAppHost>
+    <CLRTestTargetUnsupported>true</CLRTestTargetUnsupported>
+    <CLRTestTargetUnsupported Condition="'$(TargetsWindows)' == 'true'">false</CLRTestTargetUnsupported>
```

Comment:
Would we expect only Windows to work with `<UseAppHost>true</UseAppHost>`, or should I try re-enabling it for the other platforms again?

Reply:
I do not think that the core runtime tests respect `UseAppHost`. The core runtime tests are run using `corerun.exe` and I do not think there is a way to override it. 

I do not see the `_AppHost` test running in the CI. Do you see it running when you run the tests locally?

Reply:
I'm not sure if they're running, but they are building on CI as can be seen by these failures: https://github.com/dotnet/runtime/runs/21308548079, https://github.com/dotnet/runtime/runs/21308441017.

Reply:
Btw, I tried https://github.com/dotnet/runtime/pull/98007/commits/99f07de0d4f7c5e0b7364d2d4e75ce7c18c0229f to see if it was running on Windows, and it didn't have the failures I'd expect (NRE). I'm trying it again on all platforms, but by specifying `DisableProjectBuild` without just copying it from `CLRTestTargetUnsupported`. This way we should (hopefully) be able to simultaneously see which platforms it doesn't work on, along with which platforms it is running on.